geometry_msgs/msg/Inertia.msg

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
In this case, the inertia tensor comment is only applied to the first element. Would it be better to describe the the tensor at the top of the message (ie. putting the comment block here)?

Reply:
The next line contains the comment `Mass [kg]` which should be associated with the field `float64 m` rather than the message level. Therefore the new empty line.

Restructuring comments from somewhere below in the file to the message level is out of scope for this PR. Please feel free to do so in a follow up PR if it seems beneficial.